examples/spark/pytorch/pytorch_lightning_spark_mnist.py

```diff
@@ -189,17 +189,34 @@ def on_train_end(self, trainer, model):
 
     callbacks = [MyDummyCallback()]
 
-    # added EarlyStopping and ModelCheckpoint
-    from pytorch_lightning.callbacks.model_checkpoint import ModelCheckpoint
-    callbacks.append(ModelCheckpoint(monitor='val_loss', mode="min",
-                                     save_top_k=1, verbose=True))
-
-    from pytorch_lightning.callbacks.early_stopping import EarlyStopping
-    callbacks.append(EarlyStopping(monitor='val_loss',
-                                   min_delta=0.001,
-                                   patience=3,
-                                   verbose=True,
-                                   mode='min'))
+    if version.parse(torch.__version__) < version.parse('1.13'):
+        """
+        torch.distributed.ReduceOp is used in ModelCheckpoint and EarlyStopping.
+        Since torch 1.13, it doesn't support condition check in Lightning code.
+        Broken line in lightning code (https://github.com/Lightning-AI/lightning/blob/master/src/pytorch_lightning/strategies/horovod.py#L179)
```

Comment:
Should we fix the lightning side, or create a Bug in lightning repo?

Reply:
I think this needs the work of fixing Horovod plugin in PTL, I will open an issue on PTL site later.